engine-tests/src/utils/mod.rs

```diff
@@ -1000,11 +1000,15 @@ pub fn unwrap_revert_slice(result: &SubmitResult) -> &[u8] {
     }
 }
 
-pub fn panic_on_fail(status: TransactionStatus) {
+pub fn panic_on_fail(status: TransactionStatus) -> Option<String> {
```

Comment:
Same comment as on other PR: https://github.com/aurora-is-near/aurora-engine/pull/947#discussion_r1716885738